package.json

```diff
@@ -67,7 +67,46 @@
           ]
         }
       ],
-      "@semantic-release/release-notes-generator",
+      [
+        "@semantic-release/release-notes-generator",
+        {
+          "preset": "conventionalCommits",
+          "parserOpts": {
+            "noteKeywords": [
+              "BREAKING CHANGE",
+              "BREAKING CHANGES",
+              "BREAKING"
+            ]
+          },
```

Comment:
Do we need to set this, as it's default?

Reply:
When using `conventionalCommits` as the preset `BREAKING` is not included by default (noted here for the default preset `angular` as well):
https://github.com/semantic-release/release-notes-generator#usage

https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog-conventionalcommits/parser-opts.js

Also, there was this note in the suggested fix: ``` (note i needed to write explicitly default values, I guess this is because it's overriding the config)```